packages/fxa-content-server/tests/functional/oauth_sign_in_token_code.js

```diff
@@ -117,27 +117,5 @@ registerSuite('OAuth signin token code', {
           )
       );
     },
-
-    'verified - valid code then click back': function () {
```

Comment:
talked through this test with @vbudhram and neither one of us quite understood its value. ripped it out 👷‍♂️

Reply:
Cool, looks like that resolved one of the failures and the other didn't happen again